package.json

```diff
@@ -1,106 +1,113 @@
 {
-  "name": "lwc",
-  "private": true,
-  "description": "Lightning Web Components",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/salesforce/lwc.git"
-  },
-  "scripts": {
-    "prepare": "yarn build",
-    "clean": "lerna run clean && lerna clean --yes && rm -rf node_modules",
-    "lint": "eslint packages/ scripts/ --ext=js,ts --rulesdir=./scripts/eslint/rules",
-    "format": "prettier --write '{packages,scripts}/**/*.{js,ts,json,md}'",
-    "build": "lerna run build --ignore perf-benchmarks --ignore integration-tests",
-    "test": "jest --config ./scripts/jest/root.config.js",
-    "test:ci": "lerna run --concurrency=1 --stream test -- --no-cache --runInBand --coverage",
-    "test:integration": "lerna exec --scope integration-tests -- yarn sauce",
-    "test:performance": "lerna exec --scope perf-benchmarks -- best --runner remote",
-    "changelog:generate": "conventional-changelog -p angular -i CHANGELOG.md -s -r 0",
-    "release:ci:npm": "./scripts/release_npm.sh",
-    "release:ci:changelog": "./scripts/release_changelog.sh",
-    "release:ci:dist-tags": "./scripts/release_dist-tags.sh"
-  },
-  "devDependencies": {
-    "@babel/core": "7.1.0",
-    "@commitlint/cli": "^6.2.0",
-    "@commitlint/config-conventional": "^6.1.3",
-    "@types/acorn": "~4.0.3",
-    "@types/jest": "~23.0.0",
-    "@types/node": "8.9.4",
-    "@typescript-eslint/eslint-plugin": "^1.3.0",
-    "@typescript-eslint/parser": "^1.3.0",
-    "aws-sdk": "~2.378.0",
-    "commitizen": "~2.9.6",
-    "concurrently": "^3.5.1",
-    "conventional-changelog-cli": "~1.3.5",
-    "cz-conventional-changelog": "~2.1.0",
-    "es5-proxy-compat": "0.21.4",
-    "eslint": "^5.13.0",
-    "execa": "^1.0.0",
-    "glob": "^7.1.2",
-    "husky": "^1.3.1",
-    "isbinaryfile": "^3.0.3",
-    "jest": "^24.0.0",
-    "jest-environment-jsdom-fifteen": "^1.0.0",
-    "lerna": "3.7.1",
-    "lint-staged": "^8.1.4",
-    "mime-types": "~2.1.21",
-    "prettier": "^1.16.4",
-    "rollup": "~1.7.4",
-    "rollup-plugin-cleanup": "~3.1.1",
-    "rollup-plugin-compat": "0.21.4",
-    "rollup-plugin-node-resolve": "^4.0.0",
-    "rollup-plugin-replace": "^2.1.0",
-    "rollup-plugin-terser": "~4.0.4",
-    "rollup-plugin-typescript": "^1.0.0",
-    "semver": "^5.5.0",
-    "ts-jest": "^24.0.0",
-    "tslib": "^1.9.3",
-    "typescript": "3.3.3"
-  },
-  "config": {
-    "commitizen": {
-      "path": "./node_modules/cz-conventional-changelog"
-    }
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged",
-      "commit-msg": "commitlint -e $GIT_PARAMS"
-    }
-  },
-  "lint-staged": {
-    "**/*.{js,ts}": [
-      "eslint --rulesdir=./scripts/eslint/rules"
+    "name": "lwc",
```

Comment:
I'm not sure why this PR includes formatting changes but could you change it back to 2 space indentation?